csrc/scheduler/reduction.cpp

```diff
@@ -708,15 +706,14 @@ std::shared_ptr<ReductionParams> outerReductionHeuristic(
 
   // Try to hit a wave by going cross reduction
   grdim = std::min(rDimAvail(), ceilDiv(device_multiprocessor_count, gidim));
-
-  // // Extend to go to target blocks, but keep 16 iterations per thread
+  // // Extend to go to target blocks
```

Comment:
nitpick: is it intended to have `// //` here?

Reply:
no, that's typo.